bin/publish.js

```diff
@@ -142,11 +142,7 @@ function assertGitIsClean() {
   }
 
   let expectedChannelBranch =
-    options.distTag === 'canary'
-      ? 'master'
-      : options.distTag === 'latest'
-      ? 'release'
-      : options.distTag;
+    options.distTag === 'canary' ? 'master' : options.distTag === 'latest' ? 'release' : options.distTag;
```

Comment:
seems like accidental reformatting, after rebasing make sure this file isn't in your change diff still

Reply:
Went away upon rebase